### DISCUSSION TITLE
登录（在新设备如何使用简悦的高级账户）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1574] [高级账户]
> 适合在 **未登录简悦 · 高级账户的浏览器使用**。（非首次绑定 / 在新设备上使用）

描述
---

> 简悦的登录方式分为两种：

1. 使用配置文件
2. 使用授权登录

![image](https://user-images.githubusercontent.com/81074/139787409-52fd14b5-5fbc-4904-9a67-0fca2fd985e1.png)

前提
---

> 确保在坚果云（或 Dropbox ） > 根目录存在 SimpRead 文件夹 → 且文件夹存在 `simpread_config.json` 文件。

![image](https://user-images.githubusercontent.com/81074/131616003-531b236b-9cc9-4d01-bf7a-a71fb6f96b6b.png)

使用配置文件
---

> 导入配置文件即可自动登录

https://user-images.githubusercontent.com/81074/139787613-8913ae9b-9b0d-41c4-8b88-7f141a60c3ee.mp4

使用授权登录
---

> 请在 https://github.com/Kenshin/simpread/discussions/1570 查看 **登录段落**

![image](https://user-images.githubusercontent.com/81074/139787147-a1fddf96-2dc0-4150-a029-8ac375dd27b2.png)

注意事项
---

> 如果你使用了同步助手，使用此方式会自动关闭个同步选项并有提示，如使用同步助手，手动开启它就可以了。（注意：需要安装同步助手）

![image](https://user-images.githubusercontent.com/81074/139785038-0b20df6e-befe-4405-b721-0c4a5d721876.png)

关联阅读
---

- [绑定高级账户](https://github.com/Kenshin/simpread/discussions/1573)

- [在新设备上登录](https://github.com/Kenshin/simpread/discussions/1574)

- [使用授权登录](https://github.com/Kenshin/simpread/discussions/1570)

- [取消授权](https://github.com/Kenshin/simpread/discussions/1575)

- [重新授权](https://github.com/Kenshin/simpread/discussions/1576)

- [找回丢失的 UID （高级账户资格）](https://github.com/Kenshin/simpread/discussions/1571)

- [找回 UID 后如何恢复高级账户资格](https://github.com/Kenshin/simpread/discussions/1958)

- [重新找到 License（兑换码）](https://github.com/Kenshin/simpread/discussions/2756)

- [安装和配置同步助手](https://github.com/Kenshin/simpread/discussions/2754)

**Comment by sjhsbhqf** [comment]
怎么绑定高级账户，文章中的链接失效了。

**Reply by Kenshin** [reply]
嗨，购买时就有文章提示的。

另外，你既然都能找到这里，应该会看到 https://github.com/Kenshin/simpread/discussions/categories/高级账户 里面对应的内容。

因为你没有说清楚是哪种方式？首次绑定？已绑定后的登录？还是什么？所以我只能给出上面的全部相关内容的链接。

PS：链接失效的问题已修复了。

**Reply by sjhsbhqf** [reply]
谢谢了，先前主要是找不到选项页，确实没想到是右键插件进入选项页，我还以为要通过官网进去

**Reply by Kenshin** [reply]
> 谢谢了，先前主要是找不到选项页，确实没想到是右键插件进入选项页，我还以为要通过官网进去

哈哈，原来如此...  😀

**Comment by WingSky2022** [comment]
请问是否支持登录超过3个用户？
因为个人设备（macbook+Windows）* （家用 + 办公 ） > 4台设备
如果再购买个License，又不知道怎么添加到同个UID里

**Reply by Kenshin** [reply]
暂时不支持多个 License 添加，但这算是 Feature，优先级不是很高。

至于你的问题，可以这样：可以使用同一个配置文件（即同一个 UID）都可以同时登录，但仅在有需要的设备上授权高级账户。

**Reply by WingSky2022** [reply]
ok，thanks，I try try. 这样也可以。
这样操作的话，不会影响到json和插件中心的配置吧？

**Reply by Kenshin** [reply]
对配置文件有影响，因为授权和非授权在配置文件中是有区别的。

插件的话，大部分都可以使用，少部分（占比 0.1 左右）需要高级账户功能。

PS：插件都是基础账户即可使用，但个别插件的功能是高级账户功能。

**Reply by WingSky2022** [reply]
好 那我还是控制一下3账户吧，谢谢

**Reply by Kenshin** [reply]
okay 毕竟不是每台设备都是主力。

**Comment by Enmber** [comment]
导入配置文件显示与本地插件不一致，需要刷新，刷新后高级账户权限仍旧是锁定状态

**Reply by Kenshin** [reply]
试试 https://www.yuque.com/kenshin/simpread/sw4m21

**Reply by Kenshin** [reply]
1. 不要用 Email 回复方式。
2. 把你的 UID 发送到 `help#####simpread.pro` 邮箱里面（注意 `#` → `@`）并注明这个 Issues ID：1574

然后请在这里告诉我（留意第一条），请进入 https://github.com/Kenshin/simpread/discussions/1574#discussioncomment-8366287 回复我。

**Reply by Enmber** [reply]
抱歉抱歉，对这个github使用不太了解，这个help@@@@@simpread.pro是邮箱地址还是什么

**Reply by Kenshin** [reply]
为了防止 Spam 😂 一个 `@`

**Reply by Enmber** [reply]
已经发送了

**Reply by Enmber** [reply]
哦，我弄好了，卸载重装后再加载配置文件就可以了
打扰了

**Reply by Kenshin** [reply]
okay 小事情，解决就好 😂

**Reply by Kenshin** [reply]
有问题请新建 https://github.com/kenshin/simpread/issues/new 讨论比较好

**Reply by Enmber** [reply]
好的，这是第一次在github上交流，一直以为这个guihub是用来下载软件的 😂

**Reply by Kenshin** [reply]
Github Issues 就是一个 QA 系统。

Github Discussions 是一个论坛系统，只不过简悦用来它做了知识库（教程 / 用法等）。
